.github/workflows/depsreview.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
```

Comment:
small nit: i think this is unnecessary (iirc)

Reply:
This is from the original action https://github.com/actions/dependency-review-action#installation

Reply:
iirc its for private repos - lets try removing and if we need to add back...

Reply:
OK, Removed it. Thanks!